src/content/docs/en/pages/devtools/cli/azion-cli-alpha/edge-application/edge-application.mdx

```diff
@@ -7,6 +7,10 @@ menu_namespace: cliMenuAlpha
 permalink: /documentation/products/cli/edge-application/
 ---
 
+:::note 
+In case one or more required fields aren't informed through the specific flags, an interactive message prompts and asks for the missing information.
```

Comment:
```suggestion
In case any required field isn't informed through the specific flags, an interactive message prompts and asks for the missing information.
```